File: Front-end/JavaScript-Fundamentals/05. Function/04. Appearance count/appearanceCount.js
function appearanceCount(input) {
    let arr,
        number,
        len,
        count = 0;

    len = +input[0];
    number = +input[2];
    arr = input[1].split(' ').map(Number);

    for (let i = 0; i < len; i += 1) {
        if (arr[i] === number) {
            count += 1;
        }
    }
    console.log(count);
}

appearanceCount(['8', '28 6 21 6 -7856 73 73 -56', '73']);

File: C Sharp/02. C# Advanced/02. Methods/Task4_AppearanceCount/AppearanceCount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Task4_AppearanceCount
{
    class AppearanceCount
    {
        static void Main()
        {
            int number = int.Parse(Console.ReadLine());
            var arr = new int[number];
            arr = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
            int x = int.Parse(Console.ReadLine());
            int counter = CountNumber(arr, x);
            Console.WriteLine(counter);
        }

        static int CountNumber(int[] example, int num)
        {
            int counter = 0;
            for (int i = 0; i < example.Length; i++)
            {
                if (example[i] == num)
                {
                    counter++;
                }
            }
            return counter;
        }
    }
}
